File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.py
def solution(num, k):
    a = []
    for i in str(num):
        a.append(int(i))
    if not str(k) in str(num):
        return -1
    for i in a:
        if i == k:
            answer = a.index(i)+1
            break
        
    return answer

File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        int answer = -1;
        char[] nums = Integer.toString(num).toCharArray();      
        for (int i = 0; i < nums.length; i++) {
            int digit = nums[i] - '0';
            if (digit == k) {
                answer = i+1;
                break;
            }
        }
        return answer;
    }
}